Source/WebKit/WebProcess/GPU/webrtc/LibWebRTCCodecs.cpp

```diff
@@ -65,16 +65,18 @@ using namespace WebCore;
 static webrtc::WebKitVideoDecoder createVideoDecoder(const webrtc::SdpVideoFormat& format)
 {
     auto& codecs = WebProcess::singleton().libWebRTCCodecs();
-    if (format.name == "H264")
+    auto codecString = String::fromUTF8(format.name.data(), format.name.length());
```

Comment:
no VP8 ?

Reply:
Not needed since libvpx is used for VP8 and is done in process.
createVideoDecoder is for HW based decoders